### PR TITLE
fix no extension mock files issue

### PR DESCRIPTION
**Pull request by filipemartinsfe**

### CODE REVIEW

File: tasks/connect.js [hunk 1]
```diff
@@ -39,7 +39,15 @@ gulp.task('connect', function () {
         },
         cors(),
         history({
-          index: config.connect.historyFallbackIndex
+          index: config.connect.historyFallbackIndex,
+          rewrites: [
```
Comment (tiagomapmarques) [inline tasks/connect.js:43]: We should have this array as the default value for a config like `historyRewrites` and then use it here like
```
rewrites: config.connect.historyRewrites
```

Comment (tiagomapmarques) [reply tasks/connect.js:43]: @filipemartinsfe

Comment (filipemartinsfe) [reply tasks/connect.js]: done @tiagomapmarques

**Comment by filipemartinsfe** [comment]
@SheepFromHeaven could you also check the PR there is a policy that 3 reviewers are needed

**Comment by SheepFromHeaven** [comment]
Done, looks nice 👌

**Pull request merged**